corehq/apps/domain/models.py

```diff
@@ -334,6 +332,9 @@ def wrap(cls, data):
             data["hr_name"] = data["slug"]
             del data["slug"]
 
+        if 'location_types' in data:
+            del data['location_types']
+
```

Comment:
Shouldn't this get manually cleaned up in the migration? If not, should probably put a comment on what is essentially a handler for legacy data.